src/card/basic/card.html

```diff
@@ -3,10 +3,10 @@
     <div ng-if="$ctrl.showFilterInHeader()" ng-include="'card/basic/card-filter.html'"></div>
     <h2 class="card-pf-title">{{$ctrl.headTitle}}</h2>
   </div>
-
-  <span ng-if="$ctrl.subTitle" class="card-pf-subtitle">{{$ctrl.subTitle}}</span>
+  <span ng-class="{'hide': $ctrl.showSpinner === 'true'}" ng-if="$ctrl.subTitle" class="card-pf-subtitle">{{$ctrl.subTitle}}</span>
 
   <div class="card-pf-body">
+    <div ng-class="{'spinner spinner-lg': $ctrl.showSpinner === 'true'}"></div>
     <div ng-transclude></div>
```

Comment:
This should not be shown if showSpinner is true. You may need to use ng-show though due to the transclude.